scripts/scil_assign_color_to_trk.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+
+from dipy.io.stateful_tractogram import Space, StatefulTractogram
+from dipy.io.streamline import load_tractogram, save_tractogram
+import numpy as np
+
+from scilpy.io.utils import (assert_inputs_exist,
+                             assert_outputs_exist,
+                             add_overwrite_arg)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description='Assign an hexadecimal RGB color to a Trackvis TRK '
+                    'tractogram. The hexadecimal RGB color should be '
+                    'formatted as 0xRRGGBB',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    p.add_argument('in_tractogram',
+                   help='Tractogram.')
+
+    p.add_argument('out_tractogram',
+                   help='Colored TRK tractogram.')
+    p.add_argument('color',
+                   help='Hexadecimal RGB color (ie. 0xRRGGBB).')
```

Comment:
On wiki, Google converter and first website, hexadecimal color are always represented as such "#51f542". The # instead of 0x. My question is, why is this different ?